### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scrollIntoView' of undefined (@esx_menu_default/html/js/app:62)

### ISSUE THREAD

**Issue by ChrisM143** [HumanTree92/esx_advancedgarage#264]
**Describe the Bug | A Clear & Concise Description of Bug**
So I go to access my purchased cars and  the menu opens and no cars are there and I get this error in F8- Uncaught TypeError: Cannot read property 'scrollIntoView' of undefined (@esx_menu_default/html/js/app:62)

**Questions**
Have you made changes to anything besides the `config.lua`?: No
Have you looked through the Closed Topics?: Yes
Have you looked through the Wiki?: Yes
Are you using a Pre-Installed ESX? If Yes who?:No 
Are you using ESX Legacy?: No  But I customized the  ESX to work with it then all of a sudden it stopped working today?
Are you using OneSync?: Yes
Linux or Windows?: Linux

**Comment by HumanTree92** [comment]
You customized ESX Legacy to work with esx_advancedgarage??? This should work with ESX Legacy without any problems. Was this script working fine before or is this a fresh install problem?

**Comment by ChrisM143** [comment]
No im not using esx legacy I edited the esx I have to work with this and it was working fine until today. But im thinking it may be something else causing my whole server to run slowly because it was working perfectly for the past like week.

**Comment by HumanTree92** [comment]
Just spitballing atm but have you tried
- Stopping Server, Clearing Server Cache, and Starting Server.
- Restart esx_advancedgarage once the error pops up and then waiting a few seconds and trying it again.

I have never seen that error get displayed unless there was another UI or Menu up when trying to access the garage.

**Comment by ChrisM143** [comment]
Ya idk the server is acting fine now but the garage is still giving issues. When I first restart the server i can access the garage fine but after like 15-20 minutes the garage breaks with that error.

**Comment by ChrisM143** [comment]
It seems to break when more people get on.

**Comment by ChrisM143** [comment]
Is there a certain esx_menu_default im supposed to use?

**Comment by ChrisM143** [comment]
Could NativeUI or NativeUI_Reloaded cause conflicts with this?

**Comment by HumanTree92** [comment]
Well u need to make sure ur using all 3 of the esx_menus. But if this script works at the start of the server but then stops working I would restart the script and see if that fixes it. However make sure no other menu is open when accessing the garage.

**Comment by ChrisM143** [comment]
> Well u need to make sure ur using all 3 of the esx_menus. But if this script works at the start of the server but then stops working I would restart the script and see if that fixes it. However make sure no other menu is open when accessing the garage.

Okay somewhat of a new update I found. I left the server running all night I just logged in and the garage will work when you load in but after sometime it just stops working. So it seems like it keeps dropping its connection to the data base.

**Comment by ChrisM143** [comment]
So now it works for some people but not everyone. Weird.

**Comment by ChrisM143** [comment]
Okay I also tried restarting the script live like you said and it breaks it the garage blips on the map disappear and you cant access the garage?

**Comment by ChrisM143** [comment]
Okay weird so when I did restart it live it broke for other people but I could still access my garage but the blips on the map were gone idk what the hell is happening this makes no sense.

**Comment by ChrisM143** [comment]
Could it be the add-on cars that I added in categories for in the regular dealership?

**Comment by HumanTree92** [comment]
> Could it be the add-on cars that I added in categories for in the regular dealership?

No. If its working for some and not others i'm not sure. It only checks the Database when someone enters the marker. So each time you enter the marker it checks the database.

**Comment by ChrisM143** [comment]
Okay heres a question could my server box not having enough CPU power basically it being over used at like 150% cause the garage script to have trouble loading the players cars for there garage?

**Comment by HumanTree92** [comment]
The garage script wouldn't cause a high CPU usage. Maybe there are too many DB Checks going on which i doubt as this has been tested with 20+ people accessing the garage at the same time and no problems. When the error pops up are you able to walk out of the marker and walk back in and try to access it again. Try that a few times and see.

**Comment by ChrisM143** [comment]
Not saying that script is causing the CPU issues I have alot of scripts so the CPU runs really high so I was wondering if that could contribute to the players cars not showing up.

**Comment by ChrisM143** [comment]
Ya idk man I even switched to the esx legacy version and I still dont have my cars show up in the garage.

**Comment by ChrisM143** [comment]
Do you have or do you know how I can change this to work with ESX 1.1-1.2?

**Comment by ChrisM143** [comment]
There is no other framework options that will work with this vehicle and garage script?

**Comment by ChrisM143** [comment]
I have tried esx Legacy but it is just too broken.

**Comment by HumanTree92** [comment]
I can see if i can make a  V1 Finale. You will just have to give me time. You can try to do it yourself. You will just have to add back a few stuff from V1 Finale. You can use this as a reference https://github.com/HumanTree92/esx_advancedgarage/commit/e48086e3d751577ca1814c50114060a3d7a33fb0

**Comment by ChrisM143** [comment]
Your older version works perfectly on my framework. But I have to use the vehiclenames.lua if I could get this older version that works to pull the names of the cars from the  database that would be perfect.

**Comment by HumanTree92** [comment]
Yeah i can see about making my release version similar to my master for ya.

**Comment by ChrisM143** [comment]
> Yeah i can see about making my release version similar to my master for ya.

I would be very appreciated that would make my life a bit easier. Basically all I wanted was for the car names to be pulled from the database so if I can have that with this older version that would be very pleasing!

**Comment by Johny879** [comment]
So is there any solutions for that error? (Uncaught TypeError: Cannot read property 'scrollIntoView' of undefined (@esx_menu_default/html/js/app:62) 
I am using ESX Legacy
